### PR TITLE
GEOPY-1420: Create docs 

### PR DESCRIPTION
**Pull request by benk-mira**
**GEOPY-1420 - Create docs**

### CODE REVIEW

File: docs/source/getting_started.rst [hunk 1]
```diff
@@ -0,0 +1,62 @@
+.. _getting_started:
+
+Getting Started
+===============
+
+I you are a Geoscience ANALYST user, you can skip this section since the installation will be bundled into
+your software and using las-geoh5 is as simple as selecting from menu.  Otherwise, read on to learn how to
+install and use via ui.json file inside or outside of ANALYST.
```
Comment (domfournier) [inline docs/source/getting_started.rst:8]: I like the idea. but it is sort of assumed that GA users won't come here. We can focus on the "external" people.

File: docs/source/getting_started.rst [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+.. figure:: /images/getting_started/Miniforge3_Setup-1.png
+    :align: center
+    :width: 200
+
+.. figure:: /images/getting_started/Miniforge3_Setup-3.png
+    :align: center
+    :width: 200
+
+Registering the Conda distribution as the default Python 3.10 interpreter is totally optional.
```
Comment (domfournier) [inline docs/source/getting_started.rst:33]:
```suggestion
Registering the Conda distribution as the default Python 3.10 interpreter is optional.
```

File: docs/source/usage.rst [hunk 3]
```diff
@@ -0,0 +1,50 @@
+.. _usage:
+
+Basic usage
+===========
+
+If you are a Geoscience ANALYST user, you can also skip this section since the import/export dialogs will be available
+through the menu system in ANALYST.  I you are not an ANALYST user, you should give it a try!  las-geoh5 will be
+included in the free viewer.  If you're *still* not convinced, or if you're an advanced user or developer and need to
+run a custom ui.json, then read on to learn how to run via ui.json.
```
Comment (domfournier) [inline docs/source/usage.rst:9]: ;) You want full-blown marketing department here. Would be tempted to leave it out, but we can let a third party decide.

File: docs/source/import.rst [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+.. figure:: /images/import/drillhole_group_and_file_selection.png
+    :align: center
+    :width: 45%
+
+    *Choosing drillhole group and files for import.*
+
+In order to organize the data by property, the user provides a name to the
+``geoh5py.property_group`` and a collocation tolerance that is used by
+*geoh5py* to determine if incoming data should be included in existing
```
Comment (domfournier) [inline docs/source/import.rst:35]:
```suggestion
`geoh5py <https://mirageoscience-geoh5py.readthedocs-hosted.com/en/stable/index.html>`_ to determine if incoming data should be included in existing
```

File: docs/source/import.rst [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+property groups with the same name.  ie: if incoming data's depth sampling
+is not within the collocation tolerance of an existing property group's
+depths and the existing group has the same name - a new property group will
+be created.
```
Comment (domfournier) [inline docs/source/import.rst:39]:
```suggestion
property groups with the same name.  A new property group is
created if the incoming depth sampling is outside of the collocation tolerance.
```

File: docs/source/import.rst [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+for initial imports, but a nuisance in subsequent import operations.
+
+.. figure:: /images/import/warnings_control.png
+    :align: center
+    :width: 15%
+
+    *Setting warnings preference.*
+
+To learn how to export data from a ``geoh5py.drillhole_group`` continue on to
+the :ref:`export` section.
```
Comment (domfournier) [inline docs/source/import.rst:81]:
```suggestion
```

**Review by domfournier — COMMENTED** [review]
Looks great. See comments

File: docs/source/usage.rst (hunk 3 above)
Comment (benk-mira) [reply docs/source/usage.rst:9]: ABC dom.  Always-be-closing!

File: docs/source/getting_started.rst (hunk 1 above)
Comment (benk-mira) [reply docs/source/getting_started.rst]: removed

File: docs/source/usage.rst (hunk 3 above)
Comment (benk-mira) [reply docs/source/usage.rst]: removed

File: docs/source/import.rst (hunk 6 above)
Comment (benk-mira) [reply docs/source/import.rst]: done

**Pull request merged**